debian/postinst

```diff
@@ -31,12 +32,12 @@ echo "configuring django database and static files"
 /usr/share/python/archivematica-storage-service/bin/python manage.py migrate
 /usr/share/python/archivematica-storage-service/bin/python manage.py collectstatic --noinput
 
-
 echo "updating directory permissions"
 chown -R archivematica:archivematica /var/archivematica/storage-service
 chown -R archivematica:archivematica /var/archivematica/storage_service
 chown -R archivematica:archivematica /var/archivematica/.storage-service
 chown -R archivematica:archivematica /usr/share/python/archivematica-storage-service
+chown -R archivematica:archivematica /var/log/archivematica
```

Comment:
I'm not sure.  There is no guarantee that Archivematica itself will get installed on the same machine as the storage service.  Set gid is probably not a good idea for the storage service, I think?  Only the one user (archivematica) should be able to write to the storage service logs..  For Archivematica the situation is a little different because of the archivematicadashboard user.  The storage service would prefer g+rs I think?  

Maybe the storage service should be logging to /var/log/archivematica/storage-service, and the dashboard to /var/log/archivematica/dashboard?  We were basing this layout on things like nginx and elasticsearch, which do not use setgid.